Code-Book/settings.py

```diff
@@ -120,6 +128,14 @@
 # https://docs.djangoproject.com/en/4.2/howto/static-files/
 
 STATIC_URL = 'static/'
+MEDIA_URL = 'images/'
+
+LOGIN_URL = '/login'
+LOGOUT_REDIRECT_URL = '/login'
```

Comment:
LOGIN_URL sends users where to the specified url in case of authentication required.
LOGOUT_REDIRECT_URL redirects the user to this url after successful logout.